static/include/app/apis/generated/data.md

```diff
@@ -135,16 +159,18 @@ You can also find your binary data under the **Images**, **Point clouds**, or **
 from viam.utils import create_filter
 from viam.proto.app.data import Filter, TagsFilter, TagsFilterType
 
-
 # Get data captured from camera components
-my_data = []
-last = None
-my_filter = create_filter(component_name="camera")
-while True:
-    data, count, last = await data_client.binary_data_by_filter(my_filter, last)
-    if not data:
-        break
-    my_data.extend(data)
+camera_filter = create_filter(
+    component_name="camera-1"
+    )
+    
+data, count, last = await data_client.binary_data_by_filter(
```

Comment:
limit=1 must be set as such if include binary data = true, I get an error saying to set it to 1 otherwise. It retrieves more than 1 piece of data, it appears to retrieve all the images

Reply:
but yes readding the loop